tests/extras/datasets/tracking/test_json_dataset.py

```diff
@@ -115,26 +115,28 @@ def test_catalog_release(self, mocker):
         data_set.release()
         fs_mock.invalidate_cache.assert_called_once_with(filepath)
 
-    def test_version_str_repr(self):
+    def test_version_str_repr(self, load_version, save_version):
         """Test that version is in string representation of the class instance
         when applicable."""
         filepath = "test.json"
-        ds_versioned = JSONDataSet(filepath=filepath)
+        ds = JSONDataSet(filepath=filepath)
+        ds_versioned = JSONDataSet(
+            filepath=filepath, version=Version(load_version, save_version)
+        )
+        assert filepath in str(ds)
+        assert "version" not in str(ds)
 
         assert filepath in str(ds_versioned)
-        ver_str = "version=Version(load=None, save=None)"
+        ver_str = f"version=Version(load={load_version}, save='{save_version}')"
         assert ver_str in str(ds_versioned)
         assert "JSONDataSet" in str(ds_versioned)
+        assert "JSONDataSet" in str(ds)
         assert "protocol" in str(ds_versioned)
+        assert "protocol" in str(ds)
         # Default save_args
+        assert "save_args={'indent': 2}" in str(ds)
         assert "save_args={'indent': 2}" in str(ds_versioned)
 
-    def test_no_versions(self, json_dataset):
-        """Check the error if no versions are available for load."""
-        pattern = r"Did not find any versions for JSONDataSet\(.+\)"
-        with pytest.raises(DataSetError, match=pattern):
-            json_dataset.load()
-
```

Comment:
Isn't this still relevant?

Reply:
`load` isn't supported for these datasets. These tests were added as a copy-paste, but don't actually make sense for these datasets imo.